api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -48,6 +48,17 @@ pub trait Rpc {
 		asset: Asset,
 		destination_address: AddressString,
 	) -> RpcResult<WithdrawFeesDetail>;
+
+	#[method(name = "encode_btc_smart_contract_call", aliases = ["broker_encodeBtcSmartContractCall"])]
+	async fn encode_btc_smart_contract_call(
+		&self,
+		output_asset: Asset,
+		output_address: AddressString,
+		retry_duration: u32,
+		min_output_amount: u128,
+		boost_fee: Option<BasisPoints>,
+		dca_parameters: Option<DcaParameters>,
+	) -> RpcResult<Vec<u8>>;
```

Comment:
Vec<u8> is not a very json-friendly return value. At the very least it should be hex-encoded, but I think we can also do better to communicate *what* it is. For example, we could make it an object with a named field. Combined with the above suggestion, a suitable output type might be:

```
pub enum SwapPayload {
	Bitcoin {
		nulldata_utxo: Bytes, // sp_core::Bytes serializes as hex
	},
	// Other variants to be added later: Eth, Sol
}
```